src/composables/list.ts

```diff
@@ -0,0 +1,49 @@
+import { DateRangeProps } from "./date"
+
+export interface DetailListConfig {
+  /**
+   * alwaysHideNav disables navigation components from displaying whatsoever.
+   * This makes the List as a "single-page" limiting the number of results via perPage.
+   */
+  alwaysHideNav?: boolean
+
+  /**
+   * date turns on the date-range filter input
+   */
+  dateSearch?: boolean | DateRangeProps
```

Comment:
```suggestion
  disableDate?: boolean
```

Looks like this prop never makes use of `DateRangeProps`.  From the current and new use cases, we can probably punt on allowing it to be configurable and revisit as needed.  I'm unsure how to think about sane defaults on the component in general, but for consistency I suppose we work from the assumption you get the kitchen sink of tools and disable what you don't need.  So this could just be `disableDate?: boolean` that defaults to false.  It feels weird, but with boolean props I'm usually aiming to keep their default to false so that you can `<DetailList disable-date />`  as opposed to `<DetailList date-search="false" />`.